### PR TITLE
Updated misc articles to fix up version numbers, and update text to …

### PR DESCRIPTION
**Pull request by OsirisTerje**
…match latest versions of NUnit

### CODE REVIEW

File: docs/articles/nunit/getting-started/dotnet-core-and-dotnet-standard.md [hunk 1]
```diff
@@ -27,10 +29,10 @@ the necessary references.
 
 ```xml
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.0" />
```
[the remark this answers is not in the document]
Comment (OsirisTerje) [reply docs/articles/nunit/getting-started/dotnet-core-and-dotnet-standard.md:30]: Sure :-)

**Comment by OsirisTerje** [comment]
I looked up more on MD013.  There are discussion on this, and the main point for readabikity is what they call  [Semantic Line Breaks](sembr.org). 
This talks about having sentences on separate lines.  
That makes more sense to me, than having arbitrary configured line lengths enforcing breaks in the sentences.  
A good issue is in markdownlint repo [#502](https://github.com/DavidAnson/markdownlint/issues/502), the reporter there express much better than me what is the reason for setting MD013 off, and replacing it with a sentence rule. 

That issue is taken further in issue [#298](https://github.com/DavidAnson/markdownlint/issues/298) which is still open, but there is a [PR there](https://github.com/markdownlint/markdownlint/pull/319) which seems to have been merged. 

That means there are mulgtiple modes for this:

```txt
Modes:

space - First space past 80 chars. Current default. Plan to deprecate.
wrapped - 80 chars, like any other linter. aliased to strict for NodeJs compat. Maybe to be the future default.
not_wrapped - No wrapping - every para is a single line.
sentence - Must break at every sentence, and no mid-sentence breaks
semantic - Must break at every sentence, allows mid-sentence breaks.
Other options:

code_blocks - same as today, enforce on codeblocks
tables - same as today, enforce on tables
exempt_single_words - Exempt single worlds.
``` 

I am not sure this covers the #502 comments,  but should do. 

If the automatic formatter @SeanKilleen mentions works with this in VS Code (and he can explain where to find i, I didn't, Format document have no effect on this) it should be better than arbitrary in-sentence breaks.

**Comment by OsirisTerje** [comment]
Thank you for offering to format stuff.  But what do we do with later editing text that have a lot of breaks in it ? 

We could separate between formatted text and non-formatted text and then use a script to do the line breaks moving the original non-formatted text into a new formatted document. That way it would be possible to edit the original text without having it broken up into pieces.  And still have formatted text passing the linter.

**Comment by SeanKilleen** [comment]
@OsirisTerje 

image

image

I ran `lint` inside the dev container and then ran that command on all offending files. Linting fixed.

**Pull request merged**